File: ComparativaRendimiento/bench.py
import time

def fibo(n):
    if n <= 1: return n
    return fibo(n-1) + fibo(n-2)

start = time.time()
res = fibo(40)
print(f"Python Result: {res} | Time: {time.time() - start:.4f} s")

File: ComparativaRendimiento/bench.c
#include <stdio.h>
#include <time.h>

long long fibo(int n) {
    if (n <= 1) return n;
    return fibo(n-1) + fibo(n-2);
}

int main() {
    clock_t start = clock();
    long long res = fibo(40);
    double time = (double)(clock() - start) / CLOCKS_PER_SEC;
    printf("C Result: %lld | Time: %.4f s\n", res, time);
    return 0;
}
